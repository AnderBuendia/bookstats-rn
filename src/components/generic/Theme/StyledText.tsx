import { Text, TextProps } from '@Components/generic/Theme/Themed';

export function MonoText(props: TextProps) {
  return (
    <Text {...props} style={[props.style, { fontFamily: 'space-mono' }]} />
  );
}
