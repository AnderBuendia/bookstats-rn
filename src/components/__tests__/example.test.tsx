import { render } from '@testing-library/react-native';
import { MonoText } from '@Components/generic/Theme/StyledText';

describe('Example Text', () => {
  it('Sum 1 + 1 equals 2', () => {
    expect(1 + 1).toEqual(2);
  });

  it(`renders correctly`, () => {
    render(<MonoText>Snapshot test!</MonoText>);

    expect(/Snapshot test!/i).toMatchSnapshot();
  });
});
