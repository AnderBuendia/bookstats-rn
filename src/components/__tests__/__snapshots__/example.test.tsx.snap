// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example Text renders correctly 1`] = `/Snapshot test!/i`;
